packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -43,7 +43,7 @@ A Global Session Policy and an authentication policy control the authentication
 
 * [Authentication policy](/docs/reference/api/policy/#authentication-policy): Determines the extra levels of authentication that you want performed before a user can access an application, such as enforcing factor requirements. Every app in your org has one authentication policy, and multiple apps can share a policy. Okta provides some preset policies with standard sign-on requirements, including a default policy automatically assigned to new apps. The default policy allows access with any two factors.
 
-  You can create an authentication policy specifically for the app or create a few policies and [share them](https://help.okta.com/okta_help.htm?type=oie&id=ext-share-auth-policy) across multiple apps. If you decide later to change an app’s sign-on requirements, you can modify its policy or switch to a different policy using the [Authentication Policies page](https://help.okta.com/okta_help.htm?type=oie&id=ext-create-auth-policy). See [Configure a Global Session Policy and an authentication policy](/docs/guides/configure-signon-policy/main/).
+  You can create an authentication policy specifically for the app or create a few policies and [share them](https://help.okta.com/okta_help.htm?type=oie&id=ext-share-auth-policy) across multiple apps. If you decide later to change an app’s sign-on requirements, you can modify its policy or switch to a different policy using the [Authentication Policies page](https://help.okta.com/okta_help.htm?type=oie&id=ext-create-auth-policy). See [Configure a Global Session Policy and an authentication policy](/docs/guides/configure-signon-policy/main/). Additionally, you can [merge duplicate authentication policies with identical rules](https://help.okta.com/okta_help.htm?type=oie&id=ext-merge-auth-policies) to improve policy management.
```

Comment:
nit: I'd be inclined to say `simply policy management` vs improve but don't feel that strongly about it